iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -3,7 +3,6 @@
     <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net5.0;netstandard2.1;netstandard2.0;net472;net451</TargetFrameworks>
     <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">net5.0;netstandard2.1;netstandard2.0</TargetFrameworks>
     <LangVersion>8.0</LangVersion>
-    <!-- TODO #176: Enable warnings as errors. -->
```

Comment:
This should have been removed back in the day when we turned warnings as errors on.